views/lib/ldap_migration_banner.hbs

```diff
@@ -1,9 +1,10 @@
-{{#if @root.currentSchoolData.officialSchoolNumber}}
-    {{#if @root.currentSchoolData.inUserMigration}}
-        {{#userHasPermission 'IMPORT_USER_VIEW'}}
-            <input type="hidden" id="inUserMigrationStarted" value="{{currentRole}}"/>
-        {{/userHasPermission}}
-    {{else}}
+
+{{#if @root.currentSchoolData.inUserMigration}}
+    {{#userHasPermission 'IMPORT_USER_VIEW'}}
+        <input type="hidden" id="inUserMigrationStarted" value="{{currentRole}}"/>
+    {{/userHasPermission}}
+{{else}}
+    {{#if @root.currentSchoolData.officialSchoolNumber}}
```

Comment:
What does this change do? There is no description in the PR and the source code is not really explicit and self-explanatory.

Reply:
In BRB you can not set the school in user migration if it does not have a school number. For other schools that is possible.
I change the IFs, so that the banner is shown if the migration is already started independently of the school number. Only to start the migration the BRB way requires the school number